pkg/kubelet/eviction/helpers.go

```diff
@@ -847,10 +849,33 @@ func makeSignalObservations(summary *statsapi.Summary) (signalObservations, stat
 	result := signalObservations{}
 
 	if memory := summary.Node.Memory; memory != nil && memory.AvailableBytes != nil && memory.WorkingSetBytes != nil {
-		result[evictionapi.SignalMemoryAvailable] = signalObservation{
-			available: resource.NewQuantity(int64(*memory.AvailableBytes), resource.BinarySI),
-			capacity:  resource.NewQuantity(int64(*memory.AvailableBytes+*memory.WorkingSetBytes), resource.BinarySI),
-			time:      memory.Time,
+		if runtime.GOOS == "windows" {
```

Comment:
Is it possible to add a unit test to cover this code? We have a lot of tests for helpers.

Reply:
I updated the existing test to change behavior when running on Windows.
This approach seemed less disruptive to the existing code and should give adequate coverage.
We do run the unit tests on Windows nodes regularly and monitor the updates - https://testgrid.k8s.io/sig-windows-signal#windows-unit-master